src/build/build1.ss

```diff
@@ -58,6 +60,9 @@
                       debug: debug optimize: optimize? generate-ssxi: gen-ssxi? static: static?
                       gsc-options: ["-cc-options" "--param max-gcse-memory=300000000"]]))
 
+(def (compile-group group . options) ;; TODO: parallelize this?
+  (parallel-build group (lambda (x) (apply compile1 x options)) false))
```

Comment:
Let's not do this! There are order dependencies between the modules, so that the optimizer info files are generated.
Otherwise we'll end up with an unoptimized version of the expander and the compiler.